src/panels/config/backup/ha-config-backup-settings.ts

```diff
@@ -211,16 +174,17 @@ class HaConfigBackupSettings extends LitElement {
   private async _save() {
     await updateBackupConfig(this.hass, {
       create_backup: {
-        agent_ids: this._backupConfig.create_backup.agent_ids,
-        include_folders: this._backupConfig.create_backup.include_folders ?? [],
-        include_database: this._backupConfig.create_backup.include_database,
-        include_addons: this._backupConfig.create_backup.include_addons ?? [],
-        include_all_addons: this._backupConfig.create_backup.include_all_addons,
-        password: this._backupConfig.create_backup.password,
+        agent_ids: this.config!.create_backup.agent_ids,
+        include_folders: this.config!.create_backup.include_folders ?? [],
+        include_database: this.config!.create_backup.include_database,
+        include_addons: this.config!.create_backup.include_addons ?? [],
+        include_all_addons: this.config!.create_backup.include_all_addons,
+        password: this.config!.create_backup.password,
       },
-      retention: this._backupConfig.retention,
-      schedule: this._backupConfig.schedule.state,
+      retention: this.config!.retention,
+      schedule: this.config!.schedule.state,
     });
+    fireEvent(this, "ha-refresh-backup-config");
```

Comment:
Maybe we should only do this on disconnected? As it will cause a lot of fetches?

Reply:
Yeah agree 🙂

Reply:
It can not be done in disconnected because it's already disconnected so the element will be fired.
Also, I don't think people will change these settings too often. 
Should I try to improve it in another way or is it ok like that ?

Reply:
It is ok for now 👍🏻 would be nice if the update call would return the new config like other APIs do